README.md

```diff
@@ -1,8 +1,8 @@
 ![Keyman](https://keyman.com/cdn/dev/img/logo2.png)
 
-Open Source
+[Keyman](https://keyman.com) makes it possible for you to type in any language on Windows, macOS, Linux, iPhone, iPad, Android tablets and phones, and even instantly in your web browser. Create keyboard layouts with Keyman Developer and share them with the community in the [keyboards repository](https://github.com/keymanapp/keyboards). The [Keyman Developer Community](https://community.software.sil.org/c/keyman) have already contributed keyboard layouts for over 1,500 languages!
 
-[Keyman](https://keyman.com) makes it possible for you to type in over 1,000 languages on Windows, iPhone, iPad, Android tablets and phones, and even instantly in your web browser.
+Keyman is an open source project distributed under the MIT license.
 
 [comment]: # (CI Build statuses)
```

Comment:
Should we go ahead and delete the table (ll. 11-18) since the dashboard on status.keyman.com is more informative?

Reply:
Yeah ... I guess ... ? It's a handy little view for the casual GitHub visitor though?